### PR TITLE
Feature femr109 diabetes prompt

### PR DESCRIPTION
**Pull request by kevinzurek**
This is a "hotfix" going directly into the master branch for the December WHSO trip.

https://teamfemr.atlassian.net/browse/FEMR-109

### CODE REVIEW

**Comment by kdunlap** [comment]
Code looks good and testing went well. I think the prompt location is perfect also. I'll go ahead and merge  this over the weekend unless I hear otherwise

**Comment by kevinzurek** [comment]
We heard otherwise:

https://groups.google.com/forum/#!topic/team-femr/K8O1I585KNs
-and-
https://groups.google.com/forum/#!topic/team-femr/ZPV4ko_oA1Y

**Comment by kevinzurek** [comment]
@kdunlap I updated this based on the new constraints. testing seems to be good so far.

Perhaps I should squash these commits so it's easier to cherry pick over to development as well?

**Comment by kevinzurek** [comment]
Added FEMR110 to this pull request to make it easier

File: public/js/medical/medical.js [hunk 1]
```diff
@@ -110,15 +110,140 @@ var multipleChiefComplaintFeature = {
...
+            //during this encounter
+            if ($('[name=isDiabetesScreenPerformed]').val() === "false") {
+                //get these values for checking BMI later
+                var heightTotalInches = recentVitals.getCurrentHeightInches() + recentVitals.getCurrentHeightFeet() * 12;
+                var weightPounds = recentVitals.getCurrentWeight();
+                var bmiScore = calculateBMIScore("standard", weightPounds, heightTotalInches);
+                //checks to see if a systolic and/or diastolic blood pressure were taken then checks to see if they
+                //surpass the threshold required for the diabetes prompt
+                if (
+                    (recentVitals.getCurrentSystolic() !== null && parseInt(recentVitals.getCurrentSystolic()) >= 140) || (recentVitals.getCurrentDiastolic() !== null && parseInt(recentVitals.getCurrentDiastolic()) >= 90)
```
Comment (kevinzurek) [inline public/js/medical/medical.js]: Needs to be changed to 135 and 80

**Comment by kdunlap** [comment]
https://teamfemr.atlassian.net/browse/FEMR-109
https://teamfemr.atlassian.net/browse/FEMR-110

**Pull request merged**